src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagename/resource/ExampleResource.kt

```diff
@@ -2,20 +2,22 @@ package uk.gov.justice.digital.hmpps.hmppstemplatepackagename.resource
 
 import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import uk.gov.justice.digital.hmpps.hmppstemplatepackagename.service.TemplateKotlinApiService
 import java.time.LocalDateTime
 
+// This controller is expected to be called from the UI - so the hmpps-template-typescript project.
 @RestController
-// Protected by a role specific to the ui - called only from the hmpps-template-typescript project
+// Role here is specific to the UI.
 @PreAuthorize("hasRole('ROLE_TEMPLATE_KOTLIN__UI')")
 @RequestMapping(value = ["/example"], produces = ["application/json"])
 class ExampleResource(private val templateKotlinApiService: TemplateKotlinApiService) {
 
   @GetMapping("/time")
   fun getTime(): LocalDateTime = LocalDateTime.now()
 
-  @GetMapping("/user-message")
-  fun getUserMessage() = templateKotlinApiService.getUserMessage()
+  @GetMapping("/user-message/{user}")
+  fun getUserMessage(@PathVariable user: String) = templateKotlinApiService.getUserMessage(user)
```

Comment:
Why do we need to call this from the UI? We've got an example above for the UI to call (/time)... if we removed this /user-message/user endpoint then we wouldn't need the example external resource, just a client and a a test that calls out to Wiremock. It's good to prove that the client works against a real endpoint, but it will just need removing along with it's associated test and exception handler, and will people remember to do that?